repair_shards.c

```diff
@@ -69,6 +72,7 @@ master_copy_shard_placement(PG_FUNCTION_ARGS)
 	ShardPlacement *targetPlacement = NULL;
 	List *ddlCommandList = NIL;
 	bool recreated = false;
+	bool dataCopied = false;
```

Comment:
We have the declaration ``ShardPlacement *sourcePlacement PG_USED_FOR_ASSERTS_ONLY = NULL;``.

Could we remove the part on ``PG_USED_FOR_ASSERTS_ONLY`` from the declaration?